worksheet_B/readme.MD

```diff
@@ -0,0 +1,38 @@
+code:
+```
+
+START
+
+Attempts=0
+i=0
+Likeness=0
+OriginalWord=Generate
+
+while (attempts <= 4):
+	Output ("enter your first attempt: ")
+	UserEntery=UserInput
+	attempts+1
+	
+	For (each character in OriginalWord):
+		i+1
+		if (letter == letter in UserEntery)
+			Likeness+1
+		else
+			next character
+		END IF
+	Output ("your likeness is: " + Likeness)
+	END FOR
+END WHILE
+
+if (Likeness == 4)
+	output ("congratulations")
+else
+	output ("you have failed")
+END IF
+
+END
+
+```
+
+
+![Flowchart](flow.png)
```

Comment:
Flowchart needs arrows to make clear the direction of flow.

It is not clear what "compare letters break after 4" means. After 4 letters? Also note that decision nodes should be diamond shaped, not rectangular.

As it stands, the user only gets one attempt at guessing the word -- another loop is needed to allow multiple attempts.